packages/flutterfire_ui/lib/src/i10n/lang/pt.dart

```diff
@@ -0,0 +1,263 @@
+    this.verifyPhoneNumberButtonText = 'Próximo',
+    this.verifyCodeButtonText = 'Verificar',
+    this.verifyingPhoneNumberViewTitle = 'Digite o código SMS',
+    this.unknownError = 'Ocorreu um erro desconhecido',
+    this.smsAutoresolutionFailedError =
+        'Falha ao detectar o código SMS automaticamente. Por favor, digite o seu código manualmente',
+    this.smsCodeSentText = 'Código SMS enviado',
+    this.sendingSMSCodeText = 'Enviando o código SMS...',
+    this.verifyingSMSCodeText = 'Verificando o código SMS...',
+    this.enterSMSCodeText = 'Digite o código SMS',
+    this.emailIsRequiredErrorText = 'O e-mail é obrigatório',
+    this.isNotAValidEmailErrorText = 'Forneça um e-mail válido',
+    this.userNotFoundErrorText = 'Conta não existe',
+    this.emailTakenErrorText = 'Já existe uma conta com esse e-mail',
+    this.accessDisabledErrorText =
+        'O acesso a esta conta foi temporariamente desativado',
+    this.wrongOrNoPasswordErrorText =
+        'A senha é inválida ou o usuário não tem uma senha',
+    this.signInText = 'Fazer login',
+    this.registerText = 'Registrar',
+    this.registerHintText = 'Não tem uma conta?',
+    this.signInHintText = 'Já tem uma conta?',
+    this.signOutButtonText = 'Sair',
+    this.phoneInputLabel = 'Número de telefone',
+    this.phoneNumberInvalidErrorText = 'Número de telefone inválido',
+    this.phoneNumberIsRequiredErrorText = 'O número de telefone é obrigatório',
+    this.profile = 'Perfil',
+    this.name = 'Nome',
+    this.deleteAccount = 'Deletar conta',
+    this.passwordIsRequiredErrorText = 'A senha é obrigatória',
+    this.confirmPasswordIsRequiredErrorText = 'Confirme sua senha',
+    this.confirmPasswordDoesNotMatchErrorText = 'As senhas não coincidem',
+    this.confirmPasswordInputLabel = 'Confirmar senha',
+    this.forgotPasswordButtonLabel = 'Esqueceu a senha?',
+    this.forgotPasswordViewTitle = 'Esqueceu a senha',
+    this.resetPasswordButtonLabel = 'Redefinir senha',
+    this.verifyItsYouText = 'Verifique se é você',
+    this.differentMethodsSignInTitleText =
+        'Use um dos seguintes métodos para fazer login',
+    this.findProviderForEmailTitleText = 'Digite seu e-mail para continuar',
+    this.continueText = 'Continuar',
+    this.countryCode = 'Código',
+    this.codeRequiredErrorText = 'O código do país é obrigatório',
+    this.invalidCountryCode = 'Código inválido',
+    this.chooseACountry = 'Escolha um país',
+    this.enableMoreSignInMethods = 'Ative mais métodos de login',
+    this.signInMethods = 'Métodos de login',
+    this.provideEmail = 'Forneça seu e-mail e senha',
+    this.goBackButtonLabel = 'Voltar',
+    this.passwordResetEmailSentText =
+        'Enviamos um e-mail para você com um link para redefinir sua senha. Por favor, verifique seu e-mail.',
+    this.forgotPasswordHintText =
+        'Forneça seu e-mail e enviaremos um link para redefinir sua senha',
```

Comment:
```suggestion
        'Digite um e-mail e lhe enviaremos um link para redefinir sua senha',
```

Reply:
What do you think, `Digite seu e-mail e nós lhe enviaremos um link para redefinir sua senha`?